Deploy-CAM.ps1

```diff
@@ -2812,6 +2909,56 @@ else {
         }
     } while ( -not $domainAdminCredential )
 
+    $radiusConfig = @{
+        enableRadiusMfa = $enableRadiusMfa
+        radiusServerHost = $radiusServerHost
+        radiusServerPort = $radiusServerPort 
+        radiusSharedSecret = $radiusSharedSecret
+    }
+    if ( ($enableRadiusMfa -eq $null) -or $enableRadiusMfa ) {
```

Comment:
Why enter the prompt area if it's already set? I think the if's here can probably be simplified.

Reply:
For each option it prompts only if it's not set

Reply:
Okay - I thought I saw a logic error here but no it looks fine.